src/Main.java

```diff
@@ -1,5 +1,184 @@
 public class Main {
-    public static void main(String[] args) {
-        System.out.println("Hello world!");
+
+    private static final Employee[] employees = new Employee[]{
```

Comment:
```suggestion
    private static final Employee[] EMPLOYEES = new Employee[]{
```
потому что STATIC и FINAL